crates/web5_cli/src/vcs.rs

```diff
@@ -49,7 +49,7 @@ impl Commands {
                     Some(i) => i.to_string(),
                     None => match &portable_did {
                         Some(p) => p.did_uri.to_string(),
-                        None => String::default(),
+                        None => panic!("either --issuer or --portable-did must be supplied"),
```

Comment:
did you mean to keep the panic in?

Reply:
yeah I think it's a slightly better error DX than what the call to `::create()` will return which is just that the issuer string can't be non-empty

remember, this is in the CLI so panic is fine, preferred actually